### PR TITLE
[CPL][DESK] Add "Themes" Tab resource data

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

JIRA issue: [CORE-14537](https://jira.reactos.org/browse/CORE-14537)

BEFORE:
before
AFTER:
after

### CODE REVIEW

**Comment by HBelusca** [comment]
@katahiromz : do you expect to implement the contents of this tab? or will it remain empty?

**Comment by katahiromz** [comment]
This is initial implementation. Emptiness is OK.

**Comment by learn-more** [comment]
Or disable the page

File: dll/cpl/desk/lang/pl-PL.rc [hunk 1]
```diff
@@ -9,6 +9,19 @@
 
 LANGUAGE LANG_POLISH, SUBLANG_DEFAULT
 
+IDD_THEMES DIALOGEX 0, 0, 246, 210
+CAPTION "Themes"
```
Comment (pithwz) [inline dll/cpl/desk/lang/pl-PL.rc]: Kompozycje

Comment (pithwz) [inline dll/cpl/desk/lang/pl-PL.rc]: CAPTION "Kompozycje"
STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
FONT 8, "MS Shell Dlg"
BEGIN
    LTEXT "Kompozycja to Tapeta, zestaw dźwięków, ikon i innych elementów, które pomogą Ci spersonalizować komputer za pomocą jednego kliknięcia.", IDC_STATIC, 5, 5, 235, 30
    LTEXT "Kompozycja:", IDC_STATIC, 5, 42, 55, 10
    COMBOBOX cmb1, 5, 52, 160, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
    PUSHBUTTON "Z&apisz jako...", psh1, 170, 52, 70, 14
    PUSHBUTTON "&Usuń", psh2, 170, 70, 70, 14

**Comment by JoachimHenze** [comment]
I also think we should disable that tab as long as it does not expose any functionality.

**Comment by katahiromz** [comment]
disabled
Disabled.

**Pull request merged**